.github/workflows/ubuntu-jruby.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     steps:
```

Comment:
Could you move those 2 jobs to ubuntu.yml now that `ruby/setup-ruby` supports jruby-head?

Reply:
See PR https://github.com/ruby/psych/pull/435.  Tests are failing, so maybe leave them separate for now?

Reply:
Right, makes sense.